### PR TITLE
Bollards mesh replacements and 8k asphalt textures

### PR DESCRIPTION
**Pull request by Ekozmaster**
Changes:
- Light bollards now have a new mesh with 1024x textures
- New 8k textures for asphalt, including intersections and crossings
- Moved asphalt texture replacements from uncle_burrito.usda to roads.usda

### CODE REVIEW

File: rtx-remix/mods/gameReadyAssets/02_materials/03_props/26A6E5758911C4FE.usda [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+            asset inputs:diffuse_texture = @../../textures/Props/26A6E5758911C4FE_albedo.dds@ (
+                colorSpace = "auto"
+                customData = {
+                    asset default = @@
+                }
+                displayGroup = "Diffuse"
+                displayName = "Albedo Map"
+                doc = "The texture specifying the albedo value and the optional opacity value to use in the alpha channel"
+                hidden = false
+            )
```
Comment (nicklllmedia) [inline rtx-remix/mods/gameReadyAssets/02_materials/03_props/26A6E5758911C4FE.usda:116]: Somewhere here should be Emissive reference

**Comment by nicklllmedia** [comment]
Alright, it didn't work for me. Neither editing 26A6E5758911C4FE.usda and mat_LightBollard.usda with emissive input didn't give any result

**Comment by adamplayer** [comment]
image
Bollard replacements did not work for me either.

**Comment by adamplayer** [comment]
image
Mesh replacements seem to work now on latest builds.

**Review by adamplayer — APPROVED** [review]
Works on more than one machine now. Good to go.

**Pull request merged**